src/standalone/Selector/MultiSelect.tsx

```diff
@@ -25,73 +25,94 @@ export interface MultiSelectorData extends SelectorData {
 	canUnselect?: (data: MultiSelectorData) => boolean | Promise<boolean>;
 }
 
-export interface MultiSelectProps<Data extends MultiSelectorData>
+export interface MultiSelectProps
 	extends Omit<
-		SelectorPropsSingleSelect<Data>,
+		BaseSelectorProps,
 		"multiSelect" | "clearable" | "onSelect" | "selected"
 	> {
 	/**
 	 * Extended selection change handler
 	 * @param data The selected data entry/entries
 	 */
-	onSelect?: (value: Data[]) => void;
+	onSelect?: (value: MultiSelectorData[]) => void;
 	/**
 	 * The currently selected values
 	 */
-	selected: Data[];
+	selected: MultiSelectorData[];
 	/**
 	 * Specify a custom component for displaying multi select items
 	 */
 	selectedEntryRenderer?: React.ComponentType<IMultiSelectEntryProps>;
 }
 
-const styles = createStyles((theme: Theme) => ({
+const styles = createStyles({
 	paperWrapper: {
 		boxShadow: "none",
-		border: `1px solid ${theme.palette.divider}`,
 	},
-	selectedEntries: {},
-}));
+	inputRoot: {
+		borderRadius: "4px 4px 0px 0px",
+	},
+	selectedEntries: {
+		border: `1px solid rgba(0, 0, 0, 0.23)`,
+		borderTop: 0,
+		borderRadius: "0px 0px 4px 4px",
+	},
+});
 
-const MultiSelect = <Data extends MultiSelectorData>(
-	props: MultiSelectProps<Data> & WithStyles
-) => {
+const MultiSelect = (props: MultiSelectProps & WithStyles) => {
 	const {
 		onLoad,
 		onSelect,
 		selected,
 		enableIcons,
-		customStyles,
 		selectedEntryRenderer,
-		disable,
+		disabled,
 		classes,
```

Comment:
combine classes from MultiSelectProps with the class names from `useStyles()` (make an util function for that)

example for "combine classes":
`props.classes` =>
```js
{
  root: "generated-styles1",
  label: "generated-styles2",
}
```
`const multiSelectStyles = useStyles()` => 
```js
{
  root: "generated-styles3"
}
```
is combined (`combineStyles(multiSelectStyles, props.classes)`):
```js
{
 root: "generated-styles3 generated-styles1",
 label: "generated-styles2"
}
```
NOTE: Class name order matters

Reply:
@TQjona  - util file added as combineClassMaps